src/layout/DefaultLayout/index.tsx

```diff
@@ -1,18 +1,25 @@
-import { ReactNode } from 'react';
+import { Outlet, useLocation } from 'react-router-dom';
 import BottomNavigation from 'components/common/BottomNavigation';
 import useMediaQuery from 'utils/hooks/useMediaQuery';
 import TopNavigation from 'components/common/TopNavigation';
+import Home from 'pages/Home';
+import cn from 'utils/ts/classNames';
+import styles from './DefaultLayout.module.scss';
 
-interface DefaultLayoutProps {
-  children: ReactNode
-}
-
-export default function DefaultLayout({ children }: DefaultLayoutProps): JSX.Element {
+export default function DefaultLayout(): JSX.Element {
   const { isMobile } = useMediaQuery();
+  const location = useLocation();
   return (
     <>
       {!isMobile && <TopNavigation />}
-      {children}
+      <Outlet />
+      <div className={cn({
+        [styles.home]: false,
+        [styles.home]: location.pathname !== '/',
+      })}
```

Comment:
cn을 쓰는 이유가 무엇인가요

Reply:
cn을 활용해 조건문을 주지 않으면 `<Outlet />` 밑에 지도가 계속 표시 되어서 사용했습니다. 혹시 다른 방법이 있나요?

Reply:
지도 전체에 absolute와 z-index를 두고, 감싸는 레이아웃에서 z-index를 한 칸 높게 두면 해결되지 않을까요?

Reply:
웹페이지 기본 배경화면에 background-color를 주지 않아서 z-index를 줘도 검색 지도가 보이게 되어 background-color를 추가하면 또 z-index 때문에 지도 페이지에선 지도가 가려지더라고요. cn의 방식으로 가야할것 같아요.

Reply:
아래처럼 줄여서 사용할 수 있을 것 같아요.
```suggestion
      <div className={cn({ [styles.home]: location.pathname !== '/' })}
```

Reply:
수정했습니다. [c6f3226](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/50/commits/c6f322676d567c0975e74acc79c7afdb5d1c323b)